### PR TITLE
[drawer] connect activities to drawer

### PR DESCRIPTION
**Pull request by braitz**
# What is it?  Why?

I created Home, Services, and LearnMore activities that extend MainActivity (so we can inherit drawer implementation and share the same template activity_main layout file)

I would have created User and TrainingAndTools activities too, but didn't want to collide with you...

# Please Review

@Chezlui @xnxky

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,14 +9,32 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name=".MainActivity"
+            android:name=".home.HomeActivity"
```
Comment (xnxky) [inline app/src/main/AndroidManifest.xml]: I understand that you are trying to do testing; But the PR should still be main activity, right?

Comment (xnxky) [inline app/src/main/AndroidManifest.xml]: I probably will change this one to loginActivity in my PR.

**Comment by xnxky** [comment]
The conventional names of layout files are content_*, corresponding to activity_*.  Other parts LGTM.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (braitz) [inline app/src/main/AndroidManifest.xml]: We do need a login page.  But we also do need some activity to manage the HOME view that has those four buttons (or whatever we discuss).

**Comment by braitz** [comment]
I agree with your comment about the layout file naming;  I'll change that...

**Pull request merged**